.nycrc

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "@istanbuljs/nyc-config-typescript",
+    "all": true,
+    "recursive": true,
+    "compact": false,
+    "useInlineSourceMaps": false,
+    "extension": [
+        ".js",
+        ".jsx",
+        ".ts",
+        ".tsx"
+    ],
+    "exclude": [
+        "**/*.config.*"
```

Comment:
What have you excluded?
BTW I suspect that node_modules could also be instrumented and this might be a reason of performance degradation. What do you think?
And maybe disable instrumentation of ``tests/**/.js``?

Reply:
node_modules not included in the instrumentation. This is the default setting for nyc.
Added the directories to the exclude: 
3rdparty
tests